src/System.IO.FileSystem/tests/Directory/Delete.cs

```diff
@@ -204,6 +205,17 @@ public void UnixShouldBeAbleToDeleteHiddenDirectory()
             Assert.False(Directory.Exists(testDir));
         }
 
+        [Fact]
+        [OuterLoop("Needs sudo access")]
+        [PlatformSpecific(TestPlatforms.Linux)]
```

Comment:
Are these tests Linux-specific or Unix-specific?  I'm wondering if this should be AnyUnix instead of Linux so that the tests will also run on macOS.